matrix_client/api.py

```diff
@@ -326,6 +338,12 @@ def get_emote_body(self, text):
             "body": text
         }
 
+    def get_notice_body(self, text):
```

Comment:
I was wondering why it was like that and opted to mimic the behaviour. I've moved the body into the `send_notice` function as it does seem a lot more reasonable.